### PR TITLE
chore: clean up experimental

### PR DESCRIPTION
**Pull request by divyenduz**
Clean up after https://github.com/prisma/prisma-examples/pull/2327

### CODE REVIEW

File: .github/scripts/check-for-update.sh [hunk 1]
```diff
@@ -57,7 +57,7 @@ while [ $i -le $count ]; do
     echo "checking $item"
 
     case "$item" in
-      *".github"*|*"experimental"*)
+      (*".github"*)
```
Comment (janpio) [inline .github/scripts/check-for-update.sh]: Are you sure this `(` belongs here if it wasn't there before?

File: .github/scripts/upgrade-all.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ dir=$(pwd)
 
 echo "$packages" | tr ' ' '\n' | while read -r item; do
   case "$item" in
-    *".github"*|*"experimental"*|*"deployment-platforms"*)
+    (*".github"*|*"deployment-platforms"*)
```
Comment (janpio) [inline .github/scripts/upgrade-all.sh]: see https://github.com/prisma/prisma-examples/pull/2339/files#r540843088

File: .github/scripts/check-for-update.sh (hunk 1 above)
Comment (divyenduz) [reply .github/scripts/check-for-update.sh]: Both are valid `sh` (checked with `shellcheck`), I think I should get rid of it though, looked up the case syntax and it doesn't need the first bracket.

Comment (divyenduz) [reply .github/scripts/check-for-update.sh]: https://devhints.io/bash (search for case/switch)

Comment (janpio) [reply .github/scripts/check-for-update.sh]: Weird syntax is weird 🤷

**Comment by janpio** [comment]
Also requested review from @ruheni as he removed the experimental examples.

**Review by janpio — APPROVED** [review]

**Pull request merged**